packages/apputils/src/sessioncontext.tsx

```diff
@@ -1319,6 +1322,7 @@ export namespace SessionContext {
 export class SessionContextDialogs implements ISessionContext.IDialogs {
   constructor(options: ISessionContext.IDialogsOptions = {}) {
     this._translator = options.translator ?? nullTranslator;
+    this._settingRegistry = options.settingRegistry || null;
```

Comment:
using the setting page works now, but checkbox dialog doesn't

`this._settingRegistry = options.settingRegistry || null;` is undefined, when the setting is unset

so I think this need to be nullSettingRegistry which I can't find